android/src/oldarch/java/com/brentvatne/exoplayer/ReactExoplayerViewManager.java

```diff
@@ -42,17 +44,17 @@ public class ReactExoplayerViewManager extends ViewGroupManager<ReactExoplayerVi
     private static final String PROP_AD_TAG_URL = "adTagUrl";
     private static final String PROP_SRC_TYPE = "type";
     private static final String PROP_DRM = "drm";
-    private static final String PROP_DRM_TYPE = "type";
+    private static final String PROP_DRM_TYPE = "drmType";
     private static final String PROP_DRM_LICENSESERVER = "licenseServer";
     private static final String PROP_DRM_HEADERS = "headers";
     private static final String PROP_SRC_HEADERS = "requestHeaders";
     private static final String PROP_RESIZE_MODE = "resizeMode";
     private static final String PROP_REPEAT = "repeat";
     private static final String PROP_SELECTED_AUDIO_TRACK = "selectedAudioTrack";
-    private static final String PROP_SELECTED_AUDIO_TRACK_TYPE = "type";
```

Comment:
these changes change the api ?!

Reply:
https://github.com/react-native-video/react-native-video/pull/3204/commits/3f12e343a78621c95c69e27dcd0bea26c4617390

Please look at this commit.
It would change api as you said.
But it's necessary.

For similar case, we can find out "selectedAudioTrack".
This naming change is needed because of CodeGen with "WithDefault".
We use WithDefault type for codegen, but with it, we cannot declare same naming of WithDefault.